TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -103,6 +110,17 @@ def get_basic_auth_password(self) -> str:
         password = client.get_secret_value(SecretId=self.auth_details["password_from_secret_arn"])
         return password["SecretString"]
 
+    def _get_sigv4_details(self, force_region=False) -> tuple[str, str]:
+        """Return the service signing name and region name. If force_region is true,
+        it will instantiate a boto3 session to guarantee that the region is not None.
+        This will fail if AWS credentials are not available.
```

Comment:
From the unit tests, it seems this isn't actually the behavior, and instead will only fail if AWS Credentials are not available AND a region is not specified in auth details